libs/frontend/feature-render-builder/src/lib/compute-builder-reusable-styles.ts

```diff
@@ -28,13 +28,18 @@ export const computeBuilderReusableStyles = (site: ISite): IRawStyleRecord => {
     accumulatedBreakpointIds.forEach((breakpointId) => {
       const bpStyle = style.breakpoints[breakpointId]
       const rawStyle = bpStyle?.[CssPseudoClass.Default] ?? {}
-      result[`.${styleId}`] = { ...rawStyle }
+      const cls = `.${styleId}`
+      const currentStyle = result[cls] ?? {}
+      result[cls] = { ...currentStyle, ...rawStyle }
```

Comment:
@abemscac Instead of replacing the style, I think we need to merge. Otherwise styles from larger breakpoints aren't applied in the builder. This change was also necessary in `getBuildPageStyle`.